src/main/java/org/scijava/batch/ModuleBatchProcessor.java

```diff
@@ -114,13 +119,27 @@ public void run() {
 			scriptModule.resolveOutput(outputKey);
 		}
 
+		String taskName = "Batch:";
+		if (scriptModule.getInfo()!=null) {
+			if (scriptModule.getInfo().getName()!=null)
+				taskName = scriptModule.getInfo().getName();
+		}
```

Comment:
Codacy marks this as "error prone": These nested if statements could be combined.

```suggestion
		if (scriptModule.getInfo() != null && scriptModule.getInfo().getName() != null) {
				taskName = scriptModule.getInfo().getName();
		}
```

Reply:
But is it really a good idea to combine in this case ? Are we sure that the `if (a != null && a.b != null)` statement will not evaluate  `a.b != null` if a is null ?

Reply:
Ok, we can combine in this case (https://stackoverflow.com/questions/5805847/checking-if-an-object-is-null-at-the-same-time-as-a-value-of-one-of-its-fields)